### DISCUSSION TITLE
Mails are not going by thunderbird

### DISCUSSION THREAD

**Discussion by 7838099724** [modoboa/modoboa#2306] [Help]
mail postfix/submission/smtpd[1548055]: NOQUEUE: reject: RCPT from unknown[47.8.32.206]: 451 4.3.5 <pandeyaaditya59@gmail.com>: Recipient address rejected: Server configuration problem; from=<systemadmin@blueappleonline.com> to=<pandeyaaditya59@gmail.com> proto=ESMTP helo=<[192.168.43.87]>
Jul  1 21:13:39 mail postfix/submission/smtpd[1548055]: lost connection after RCPT from unknown[47.8.32.206]

**Comment by tonioo** [comment]
Do you see anything more in logs? Can you check if the policy daemon is started?

**Comment by H-A-L-I-S** [comment]
rejected: Server configuration problem

Tells all the story :)
